src/supernode/src/main/java/com/alibaba/dragonfly/supernode/service/cdn/Downloader.java

```diff
@@ -231,6 +231,7 @@ private HttpURLConnection createConn(String taskId, String fileUrl, String[] hea
                     }
                     if (code == checkCode) {
                         fileMetaDataService.updateLastModified(taskId, conn.getLastModified());
+                        fileMetaDataService.updateETag(taskId, conn.getHeaderField("ETag"));
```

Comment:
This could be set together with `lastModified` because of they are used together to do the same thing always.

Reply:
Thanks, this is much better. I have updated the code, would you please review again?